api-2.1/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationInterpretationTranslatorImpl_2_1.java

```diff
@@ -0,0 +1,124 @@
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case LOW:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("L", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case OFF_SCALE_LOW:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("<", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case OFF_SCALE_HIGH:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding(">", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case SIGNIFICANT_CHANGE_DOWN:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("D", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case SIGNIFICANT_CHANGE_UP:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("U", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case RESISTANT:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("R", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case SUSCEPTIBLE:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("S", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case INTERMEDIATE:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("I", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case POSITIVE:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("POS", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case NEGATIVE:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("NEG", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			case VERY_SUSCEPTIBLE:
+				interpretation.setCoding(Collections.singletonList(new Coding()
+				        .setSystem(FhirConstants.VERY_SUSCEPTIBLE_SYSTEM_URI).setCode("VS").setDisplay("Very Susceptible")));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
+				break;
+			default:
+				return interpretation;
+			
+		}
+		return interpretation;
+	}
+	
+	@Override
+	public Obs toOpenmrsType(Obs openmrsObs, CodeableConcept interpretation) {
+		openmrsObs.setInterpretation(Obs.Interpretation.valueOf(interpretation.getText()));
```

Comment:
This actually needs to invert the `toFhirResource()` method. So here we expect a coding of, e.g., "NEG" and want to translate that into `Obs.Interpretation.NEGATIVE`